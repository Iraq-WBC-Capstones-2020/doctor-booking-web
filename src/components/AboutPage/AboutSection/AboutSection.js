import React from 'react';
import { Container, Row, Col } from 'react-bootstrap';
import townImg from './imgs/undraw_quite_town_mg2q1.svg';
import clockImg from './imgs/clockImg.svg';
import searchImg from './imgs/searchImg.svg';
import { useTranslation } from 'react-i18next';

function AboutSection() {
  const { t } = useTranslation(['translation', 'validation']);
  return (
    <Container className="section-container">
      <Row className="align-items-center justify-content-center row-container">
        <Col className="text-center text-md-left" md="4">
          <h2 className="section-header">
            {t('validation:aboutPage.aboutUs.headLine')}
          </h2>
          <p>{t('validation:aboutPage.aboutUs.paragraph')} </p>
        </Col>
        <Col className="offset-md-1 d-flex justify-content-center" md="6">
          <img className="img-fluid" src={townImg} alt="town" />
        </Col>
      </Row>

      <Row className="align-items-center justify-content-center row-container">
        <Col
          className="offset-md-1 order-md-12 text-center text-md-left"
          md="6"
        >
          <h2 className="section-header">
            {t('validation:aboutPage.saveTime.headLine')}
          </h2>
          <p>{t('validation:aboutPage.saveTime.paragraph')}</p>
        </Col>
        <Col className="order-md-1 d-flex justify-content-center" md="5">
          <img className="img-fluid" src={clockImg} alt="clock" />
        </Col>
      </Row>

      <Row className="align-items-center justify-content-center row-container">
        <Col className="text-center text-md-left" md="4">
          <h2 className="section-header ">
            {t('validation:aboutPage.findDoctor.headLine')}
          </h2>
          <p>{t('validation:aboutPage.findDoctor.paragraph')}</p>
        </Col>
        <Col className="offset-md-1 d-flex justify-content-center" md="6">
          <img className="img-fluid" src={searchImg} alt="search" />
        </Col>
      </Row>
    </Container>
  );
}

export default AboutSection;
