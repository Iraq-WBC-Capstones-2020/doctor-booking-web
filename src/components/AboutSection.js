import React from 'react';

function AboutSection() {
  return <div></div>;
}

export default AboutSection;
