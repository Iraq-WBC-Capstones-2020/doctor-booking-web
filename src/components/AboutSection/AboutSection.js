import React from 'react';
import { Container, Row, Col } from 'react-bootstrap';
import townImg from './imgs/undraw_quite_town_mg2q1.svg';
import clockImg from './imgs/clockImg.svg';
import searchImg from './imgs/searchImg.svg';

function AboutSection() {
  return (
    <Container className="section-container">
      <Row className="align-items-center justify-content-center row-container">
        <Col className="text-center text-md-left" md="4">
          <h2 className="section-header">About us</h2>
          <p>
            We are a team worked on this website to solve multiple problems the
            people face when they need to go to a doctor{' '}
          </p>
        </Col>
        <Col className="offset-md-1 d-flex justify-content-center" md="6">
          <img className="img-fluid" src={townImg} alt="town" />
        </Col>
      </Row>

      <Row className="align-items-center justify-content-center row-container">
        <Col
          className="offset-md-1 order-md-12 text-center text-md-left"
          md="6"
        >
          <h2 className="section-header">Save your time</h2>
          <p>
            Save the time you always waste at clinics waiting rooms you just
            need to book the time that suits your schedule by one click
          </p>
        </Col>
        <Col className="order-md-1 d-flex justify-content-center" md="5">
          <img className="img-fluid" src={clockImg} alt="clock" />
        </Col>
      </Row>

      <Row className="align-items-center justify-content-center row-container">
        <Col className="text-center text-md-left" md="4">
          <h2 className="section-header ">Find your doctor</h2>
          <p>
            Search for a doctor by speciality or by name or just explore the
            doctors and find the one that suites your need
          </p>
        </Col>
        <Col className="offset-md-1 d-flex justify-content-center" md="6">
          <img className="img-fluid" src={searchImg} alt="search" />
        </Col>
      </Row>
    </Container>
  );
}

export default AboutSection;
