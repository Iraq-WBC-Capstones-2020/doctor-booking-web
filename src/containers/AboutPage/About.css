.section-container h2 {
  color: #026fc2;
}

.section-container p {
  color: #5d646a;
}

.row-container {
  margin-top: 6rem;
}

.row-container img {
  margin-top: 1rem;
}

.section-header {
  margin-bottom: 1rem;
  font-weight: 600;
}

.card-image {
  max-width: -webkit-max-content;
  max-width: -moz-max-content;
  max-width: max-content;
  -ms-flex-item-align: center;
      -ms-grid-row-align: center;
      align-self: center;
}

.goal-card {
  max-width: 18rem;
  border: 0;
  border-radius: 12px;
  -webkit-box-shadow: 0px 6px 16px rgba(40, 41, 61, 0.08);
          box-shadow: 0px 6px 16px rgba(40, 41, 61, 0.08);
  -webkit-transition: all 0.3s ease-in-out;
  transition: all 0.3s ease-in-out;
}

.goal-card:hover {
  -webkit-box-shadow: 0px 8px 26px rgba(40, 41, 61, 0.1);
          box-shadow: 0px 8px 26px rgba(40, 41, 61, 0.1);
}

.card-title {
  font-size: 28px;
  color: #026fc2;
  font-weight: 600;
}

.section-title {
  font-size: 32px;
  color: #026fc2;
  font-weight: 600;
}
/*# sourceMappingURL=About.css.map */