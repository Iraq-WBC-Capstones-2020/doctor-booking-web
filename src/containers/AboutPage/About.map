{
    "version": 3,
    "mappings": "AACA,AACE,UADQ,CACR,EAAE,CAAC;EACD,KAAK,ECHO,OAAO;CDIpB;;AAHH,AAIE,UAJQ,CAIR,CAAC,CAAC;EACA,KAAK,ECLG,OAAO;CDMhB",
    "sources": [
        "AboutSection.scss",
        "../../styles/_variables.scss"
    ],
    "names": [],
    "file": "AboutSection.css"
}