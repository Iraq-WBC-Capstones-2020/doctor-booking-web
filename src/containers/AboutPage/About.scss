@import '../../styles/variables';
.container {
  h2 {
    color: $primaryColor;
  }
  p {
    color: $textColor;
  }
}
