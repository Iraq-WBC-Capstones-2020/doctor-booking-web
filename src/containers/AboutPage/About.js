import React from 'react';
import AboutSection from '../../components/AboutSection';
function About() {
  return (
    <>
      <AboutSection />
    </>
  );
}

export default About;
