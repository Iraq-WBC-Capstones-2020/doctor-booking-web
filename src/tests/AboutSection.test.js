import React from 'react';
import { render } from '@testing-library/react';
import AboutSection from '../components/AboutSection/AboutSection';
import renderer from 'react-test-renderer';
import About from '../containers/AboutPage/About';

test('About section renders', () => {
  render(<AboutSection />);
});

test('About section renders correctly', () => {
  const tree = renderer.create(<AboutSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
