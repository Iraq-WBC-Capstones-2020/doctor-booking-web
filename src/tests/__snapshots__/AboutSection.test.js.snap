// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About section renders correctly 1`] = `
<div
  className="section-container container"
>
  <div
    className="align-items-center justify-content-center row-container row"
  >
    <div
      className="text-center text-md-left col-md-4"
    >
      <h2
        className="section-header"
      >
        validation:aboutPage.aboutUs.headLine
      </h2>
      <p>
        validation:aboutPage.aboutUs.paragraph
         
      </p>
    </div>
    <div
      className="offset-md-1 d-flex justify-content-center col-md-6"
    >
      <img
        alt="town"
        className="img-fluid"
        src="undraw_quite_town_mg2q1.svg"
      />
    </div>
  </div>
  <div
    className="align-items-center justify-content-center row-container row"
  >
    <div
      className="offset-md-1 order-md-12 text-center text-md-left col-md-6"
    >
      <h2
        className="section-header"
      >
        validation:aboutPage.saveTime.headLine
      </h2>
      <p>
        validation:aboutPage.saveTime.paragraph
      </p>
    </div>
    <div
      className="order-md-1 d-flex justify-content-center col-md-5"
    >
      <img
        alt="clock"
        className="img-fluid"
        src="clockImg.svg"
      />
    </div>
  </div>
  <div
    className="align-items-center justify-content-center row-container row"
  >
    <div
      className="text-center text-md-left col-md-4"
    >
      <h2
        className="section-header "
      >
        validation:aboutPage.findDoctor.headLine
      </h2>
      <p>
        validation:aboutPage.findDoctor.paragraph
      </p>
    </div>
    <div
      className="offset-md-1 d-flex justify-content-center col-md-6"
    >
      <img
        alt="search"
        className="img-fluid"
        src="searchImg.svg"
      />
    </div>
  </div>
</div>
`;
